### PR TITLE
feat: KodaDot blog

### LINKED ISSUE #6419: Blog template and few posts
Let's prepare a template for the blog layout and posts. 

The point is to have **kodadot.xyz/blog/article-name** for SEO

Let's use
- https://nuxt.com/modules/content

I'll drop the content in a separate issue soon. 
- < _ >

Generally, I think the structure of posts would be like
- headline h1
- subtitle h2 
- banner
- date (for crawl bots)
- text

Most of the stuff will be just static, so only moving a little often.

Design
---
https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1296%3A18906&mode=design&t=hKFpfsqkFJAtI1v7-1

image
image

### LINKED ISSUE #6432: [Blogpost] What is ERC6551 - Token Bound Accounts (TBA) ?
Related
- #6419 

date
---
18/3/2023

Text
---

h1
---
What is ERC6551 - Token Bound Accounts (TBA) ?

subtitle
---
Understand ERC6551 which redefines how ERC-721 tokens interact with the Ethereum blockchain by giving every non-fungible token its very own smart contract account.

h2
---
What is ERC6551?

p
---
EIP6551 is a proposal made to extend the functionality of every ERC-721 token by attaching them to specific smart contract accounts, effectively providing on-chain identity to these tokens. Each non-fungible token gets a unique, global identifier which gives it a distinct identity.

h2
---
How does ERC6551 operate?

EIP6551 operates by creating uniquely, deterministically addressed smart contract accounts for each ERC-721 token via a permissionless registry. It maintains complete compatibility with existing ERC-721 token contracts, therefore not requiring any changes to previously deployed ERC-721 contracts. Control over each token-bound account is given to the owner of the ERC-721 token, enabling the owner to initiate on-chain actions on behalf of their token.

h2
---
Unique features of ERC6551

The unique feature of ERC6551 lies in its ability to turn each ERC-721 token into a standalone entity capable of owning assets and interacting with applications on the Ethereum blockchain. This expands the functionality of non-fungible tokens, allowing them to accumulate assets and record transaction history.

h2
---
Conclusion

ERC6551 is a breakthrough in enhancing the capabilities of non-fungible tokens. Through ERC6551, ERC-721 tokens not only represent a unique digital asset but can also interact independently with the blockchain and other on-chain applications. This opens a plethora of possibilities in the world of non-fungible tokens on the KodaDot platform.
​

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #6432
- [x] Closes #6419

## Screenshot 📸

- [x] My fix has changed UI

Screenshot 2023-07-24 163223
Screenshot 2023-07-24 163208

### <samp>🤖 Generated by Copilot at b46e2e6</samp>

This pull request adds a blog feature to the NFT gallery website, using markdown files as the source of content. It introduces a new module for loading markdown files as Vue components, and updates some dependencies to improve performance and compatibility. It also adds some components and pages for rendering the blog posts and the blog index, and some markdown files for the blog content.

### <samp>🤖 Generated by Copilot at b46e2e6</samp>

> _We write the content in markdown_
> _We load it with frontmatter_
> _We render the blog with Vue_
> _We set the meta for SEO_

### CODE REVIEW

File: pages/first-time.vue
Comment (preschian) [inline pages/first-time.vue:1]: is it safe to remove this component? I couldn't find the entry point to visit this page

Comment (vikiival) [reply pages/first-time.vue:1]: I would check readme of this repo or hello.kodadot.xyz

**Review by vikiival — APPROVED** [review]
ooo

File: pages/first-time.vue
Comment (preschian) [reply pages/first-time.vue:1]: seems like no page linked to `kodadot.xyz/first-time` https://github.com/search?q=org%3Akodadot%20first-time&type=code
CMIIW

**Comment by prury** [comment]
- white text on dark mode: 




- i think this part of the text should not be bold




- mobile looking good

**Comment by exezbcz** [comment]
Hello, hello, nice job! 

Few things: 

# Landing 
image
- there is line separating the img and the rest

image
- this size 16px, please - I have updated the component there
image

heading of the article to 31px 
image


- good job with the mobile version! 👍 

# single blog
image
- subtitle to 25px, regular weight 
image
- paragraph text to 20px, line height 30px, color accentlight2 dark (grey shade of pink) - #363234
- round img with stroke on this page as well 
image

# dark mode 
- sorry that I left it out. Here it is 
image
- updated in the handoff file


thanks!

File: pages/first-time.vue
Comment (yangwao) [reply pages/first-time.vue:1]: I guess so for now! We will remake it to blog in new form I bet.

**Comment by yangwao** [comment]
I guess so for now! 
We will remake it to blog in new form, for general onboarding article for developers

https://kodadot.xyz/first-time/

**Comment by yangwao** [comment]
once this merged, this would be good to release
- https://deploy-preview-6461--koda-canary.netlify.app/blog/what-is-erc6551

**Comment by yangwao** [comment]
I've got approval from @exezbcz and will do another round as we need surf erc6551

**Pull request merged**
